sonic-chassisd/scripts/chassisd

```diff
@@ -605,58 +612,153 @@ class ConfigManagerTask(ProcessTaskBase):
 
             self.config_updater.module_config_update(key, admin_state)
 
+
+#
+# State Manager task ========================================================
+#
+
+class DpuStateUpdater(logger.Logger):
+
+    DP_STATE = 'dpu_data_plane_state'
```

Comment:
How about state_reason?  Any indication of why the CP/DP is down etc?

Reply:
This information is available in the `show system-health detail` command output on the DPU. I see no reason to duplicate it.

Reply:
Having a summary of any container/service/hw gradually degrading on the DPU is good to have on the NPU. Even when the DPU is not reachable having some data like this will be very helpful for debug.

Reply:
I don't agree. Based on my experience if the data plane, control plane, or both are in a permanent down state, the summary won’t help in understanding the root cause of the problem. The user will need to connect to the DPU, read logs, check dumps, and debug the issue.

If the state change was temporary, this also won’t help in understanding the problem because the reason will be overwritten when the state changes.

Reply:
No worries.  You can commit this.

Reply:
The PMON HLD was primarily based on https://github.com/sonic-net/SONiC/blob/6edaef09c23cdc37c15983ac4711a9e5ed304d6e/doc/pmon/pmon-chassis-design.md.  Please refer to 3.2.4